core/src/main/java/hudson/util/PluginServletFilter.java

```diff
@@ -142,5 +144,25 @@ public void destroy() {
         list.clear();
     }
 
+    @Restricted(NoExternalUse.class)
+    public static void cleanUp() {
+        PluginServletFilter instance = getInstance(Jenkins.getInstance().servletContext);
+        if (instance != null) {
+            for (Iterator<Filter> iterator = instance.list.iterator(); iterator.hasNext(); ) {
```

Comment:
Because what a servlet container is supposed to do is this try-catch and we are running that ourselves... arguably the `destroy()` method above should also be doing this in a try-catch, but that is a separate improvement IMHO